ci/unit_tests/dialog_json2xml/main_data/expectedNodeTypesValid.xml

```diff
@@ -0,0 +1,28 @@
+<nodes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
I think that it will be better to just point on existing schema, not duplicating it.

Reply:
You are right.